service/src/main/java/tds/assessment/AssessmentServiceApplication.java

```diff
@@ -2,8 +2,12 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Import;
+
+import tds.common.configuration.DataSourceConfiguration;
 
 @SpringBootApplication
+@Import(DataSourceConfiguration.class)
```

Comment:
Instead of importing here please add it to the imports in ConfigServiceApplicationConfiguration.  Each support service now has one of these that centralizes imports and any bean configurations that don't fit under a specific use.